tests/core/daemon/test_daemon.py

```diff
@@ -29,9 +29,14 @@ async def get_daemon(self, get_b_tools):
     # because of a hack in shutting down the full node, which means you cannot run
     # more than one simulations per process.
     @pytest_asyncio.fixture(scope="function")
-    async def simulation(self, get_b_tools, get_b_tools_1):
+    async def simulation(self, bt, get_b_tools, get_b_tools_1):
```

Comment:
I'm pretty sure `get_b_tools` is currently referring to the global `bt` object (and you could replace that with the new fixture. However, `get_b_tools_1` should be a copy of `bt`. i.e. the same kind of fixture but a separate instance.

Reply:
Yes, that was my impression - `get_b_tools_1` is safe as-is for now, correct?